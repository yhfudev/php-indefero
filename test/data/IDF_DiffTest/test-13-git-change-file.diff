diff --git a/b b/b
index bfdaa0f..e528a92 100644
--- a/b
+++ b/b
@@ -1,3 +1,4 @@
 a
+l
 b
 
