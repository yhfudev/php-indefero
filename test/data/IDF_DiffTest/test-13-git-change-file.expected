<?php return array (
  'b' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '1',
          2 => "a\n",
        ),
        1 => 
        array (
          0 => '',
          1 => 2,
          2 => "l\n",
        ),
        2 => 
        array (
          0 => 2,
          1 => 3,
          2 => "b\n",
        ),
        3 => 
        array (
          0 => 3,
          1 => 4,
          2 => "\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '3',
        ),
        1 => 
        array (
          0 => '1',
          1 => '4',
        ),
      ),
    ),
  ),
);